include/validation_functions.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+function isEmpty($dataReturned){                            
+   $data = [];
+   
+    if($dataReturned == NULL || $dataReturned == ''){
+        $data['rows'] = 'Empty';                                    
```

Comment:
This is nitpicky, but I think it might be easier to return a boolean here. This isn't wrong at all, just that booleans make it a little easier to do if/else statements. If you need access to the string here, then feel free to leave it as-is, but just wanted to mention it!

Reply:
Yes! I agree! I changed. I might use an array when validating errors but I don't think I will use the array here.

Reply:
I also added a question as a comment in the code. Is that ok? Or should I add those comments here?

Reply:
Not to be too redundant, but either works. I get notifications about conversations in GH so that at least will let me know you have something to talk about, but feel free to go with either approach.